build.sbt

```diff
@@ -1,4 +1,4 @@
-import io.gatling.build.SonatypeReleasePlugin
+//import io.gatling.build.SonatypeReleasePlugin
```

Comment:
I commented out the lines instead of removing completely for clarity. We won't support this code, it's temporarily here until official Gatling 3 is released.